cscs-checks/microbenchmarks/simd/vc.py

```diff
@@ -0,0 +1,82 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+@rfm.simple_test
+class VcSimdTest(rfm.RegressionTest):
+    """
+    Testing https://github.com/VcDevel/Vc
+    Example performance report will look like:
+    > reframe --system dom:mc -p PrgEnv-gnu -r -c vc.py --performance-report
+        PERFORMANCE REPORT
+        -----------------------------------------------------------------------
+        VcSimdTest
+        - dom:mc
+           - PrgEnv-gnu
+              * speedup: 1.3813 cyc
+    > reframe --system dom:gpu -p PrgEnv-gnu -r -c vc.py --performance-report
+        PERFORMANCE REPORT
+        -----------------------------------------------------------------------
+        VcSimdTest
+        - dom:gpu
+           - PrgEnv-gnu
+              * speedup: 1.3824 cyc
+    """
+    def __init__(self):
+        self.valid_systems = ['*']
+        self.valid_prog_environs = ['PrgEnv-gnu']
+        self.descr = 'finitediff example'
+        self.build_system = 'SingleSource'
+        self.testname = 'finitediff'
+        src_url = 'https://raw.githubusercontent.com/VcDevel/Vc/1.4/examples'
+        src1 = '%s/tsc.h' % src_url
+        src2 = '%s/finitediff/main.cpp' % src_url
+        self.prebuild_cmd = [
+            'wget %s' % src1,
+            'wget %s' % src2,
```

Comment:
I think that the following should work:

```suggestion
            'wget %s %s' % (src1, src2),
```